open-craft/edx-platform
# [BB-1494] Add French translation to the registration page text

This PR adds some missing French translations to the registration page text as provided by the client.

**Testing instructions**:
* Checkout the source branch of this PR.
* Restart LMS.
* Navigate to the registration page and confirm that the translations added in this PR show up.
